### ISSUE TITLE
Add scrolling to the simulator in RocketSim

### ISSUE THREAD

**Issue by funky-monkey** [AvdLee/RocketSimApp#290]
For years I've been frustrated that I can't scroll using the touchpad or magic mouse in the simulator. 
I just [found this](https://github.com/aheze/Squirrel) and I'd love to see something similar in RocketSim.
Can it be done Twan?

**Comment by AvdLee** [comment]
Yes, I've actually got approval to implement it: https://twitter.com/aheze0/status/1621402828454629377?s=20
Great idea 🙏
